spyder/plugins/completion/languageserver/widgets/status.py

```diff
@@ -25,25 +25,44 @@
 logger = logging.getLogger(__name__)
 
 
+# Main constants
+class ClientStatus:
```

Comment:
@ccordoba12 maybe handling the dict with status translation in this class could make things easier to follow. Maybe the `ClientStatus` class could look something like this:

```python
class ClientStatus:
    STATUS = 'LSP {}: {}'
    STARTING = 'starting'
    READY = 'ready'
    RESTARTING = 'restarting'
    DOWN = 'down'
    STRINGS_FOR_TRANSLATION = {
        STARTING: _("starting"),
        READY: _("ready"),
        RESTARTING: _("restarting"),
        DOWN: _("down")
    }
```

Reply:
Good suggestion @dalthviz!